src/Map.php

```diff
@@ -794,14 +794,27 @@ public function offsetUnset($key)
     // implements \IteratorAggregate
 
     /**
-     * Get an iterator for a copy of the map.
+     * Get an iterator for the map.
      *
-     * @return \ArrayIterator
+     * @return \Traversable
```

Comment:
I'd make this `@return \Generator`, per the [phpDocumentor](https://github.com/phpDocumentor/fig-standards/issues/5) discussion on `yield`.

Notwithstanding, this is, technically, a breaking change, so the version bump is 1.0 to 2.0. That seems... extreme... for this one change. It's hard for me to envision a scenario where a caller depended upon `getIterator` returning an `ArrayIterator`, so I'm disinclined to go to 2.0. But, I'm also disinclined to break SemVer rules. Thoughts?

Reply:
A caller shouldn't be using `getIterator` directly so I don't think a major version change is necessary to be honest.

Reply:
A caller shouldn't be using `getIterator` directly so I don't think a major version change is necessary to be honest.